auth-mailchimp-sync/extension.yaml

```diff
@@ -23,7 +23,7 @@ description:
 license: Apache-2.0
 billingRequired: true
 sourceUrl: https://github.com/firebase/extensions/tree/master/auth-mailchimp-sync
-releaseNotesUrl: https://github.com/firebase/extensions/releases
+releaseNotesUrl: hhttps://github.com/firebase/extensions/blob/master/auth-mailchimp-sync/CHANGELOG.md
```

Comment:
should only be one h in https :-)

Reply:
Woops, good catch!

Reply:
no probs - thank goodness for the green highlight or I woulda missed it too! :-D